components/o-message/src/tsx/message.tsx

```diff
@@ -0,0 +1,84 @@
+import { Fragment } from 'react';
+
+type MessageContent = {
+	detail: string;
+	hightLight?: string;
+	additionalInfo?: string;
+}
+
+type MessageAction = {
+	copy: string;
+	url: string;
+	openInNewWindow?: boolean
+}
```

Comment:
```suggestion
type MessageAction = {
	label: string;
	url: string;
	openInNewWindow?: boolean
}
```

i think we've used the name `label` in other components for this